modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/PassThroughHttpSender.java

```diff
@@ -556,8 +556,8 @@ public void submitResponse(MessageContext msgContext)
                 if (contentTypeInMsgCtx != null) {
                    String contentTypeValueInMsgCtx = contentTypeInMsgCtx.toString();
                    // Skip multipart/related as it should be taken from formatter.
-                   if (!contentTypeValueInMsgCtx.contains(
-                           PassThroughConstants.CONTENT_TYPE_MULTIPART_RELATED)) {
+                   if (!contentTypeValueInMsgCtx.contains(PassThroughConstants.CONTENT_TYPE_MULTIPART_RELATED) ||
```

Comment:
this should be corrected as below
if(!(contentTypeValueInMsgCtx.contains(
                          PassThroughConstants.CONTENT_TYPE_MULTIPART_RELATED) || contentTypeValueInMsgCtx.contains(PassThroughConstants.CONTENT_TYPE_MULTIPART_FORM_DATA)))